src/test/regress/sql/multi_alter_table_add_constraints_without_name.sql

```diff
@@ -37,7 +37,83 @@ SELECT con.conname
 \c - - :master_host :master_port
 ALTER TABLE AT_AddConstNoName.products DROP CONSTRAINT products_pkey;
 
-ALTER TABLE AT_AddConstNoName.products ADD PRIMARY KEY(product_no);
+-- Check "ADD UNIQUE"
+ALTER TABLE AT_AddConstNoName.products ADD UNIQUE(product_no);
+
+SELECT con.conname
+    FROM pg_catalog.pg_constraint con
+      INNER JOIN pg_catalog.pg_class rel ON rel.oid = con.conrelid
+      INNER JOIN pg_catalog.pg_namespace nsp ON nsp.oid = connamespace
+              WHERE rel.relname = 'products';
+
+-- Check that UNIQUE constraint name created on the coordinator is sent to workers and
+-- the constraints created for the shard tables conform to the <conname>_shardid scheme.
+\c - - :public_worker_1_host :worker_1_port
+SELECT con.conname
+    FROM pg_catalog.pg_constraint con
+      INNER JOIN pg_catalog.pg_class rel ON rel.oid = con.conrelid
+      INNER JOIN pg_catalog.pg_namespace nsp ON nsp.oid = connamespace
+                WHERE rel.relname = 'products_5410000';
+
+\c - - :master_host :master_port
+ALTER TABLE AT_AddConstNoName.products DROP CONSTRAINT products_product_no_key;
+
+-- Check "ADD UNIQUE" with column name list
+ALTER TABLE AT_AddConstNoName.products ADD UNIQUE(product_no,name);
+SELECT con.conname
+    FROM pg_catalog.pg_constraint con
+      INNER JOIN pg_catalog.pg_class rel ON rel.oid = con.conrelid
+      INNER JOIN pg_catalog.pg_namespace nsp ON nsp.oid = connamespace
+              WHERE rel.relname = 'products';
+
+\c - - :public_worker_1_host :worker_1_port
+SELECT con.conname
+    FROM pg_catalog.pg_constraint con
+      INNER JOIN pg_catalog.pg_class rel ON rel.oid = con.conrelid
+      INNER JOIN pg_catalog.pg_namespace nsp ON nsp.oid = connamespace
+                WHERE rel.relname = 'products_5410000';
+
+\c - - :master_host :master_port
+ALTER TABLE AT_AddConstNoName.products DROP CONSTRAINT products_product_no_name_key;
+
+-- Check "ADD UNIQUE ... INCLUDE"
+ALTER TABLE AT_AddConstNoName.products ADD UNIQUE(product_no) INCLUDE(price);
+
+SELECT con.conname
+    FROM pg_catalog.pg_constraint con
+      INNER JOIN pg_catalog.pg_class rel ON rel.oid = con.conrelid
+      INNER JOIN pg_catalog.pg_namespace nsp ON nsp.oid = connamespace
+              WHERE rel.relname = 'products';
+
+\d+ AT_AddConstNoName.products;
```

Comment:
Let's avoid using `\d+` as it may change with different PG versions in the future. It prints a lot of details. Can you provide the same functionality here by querying a `pg_catalog` table to print the properties you are interested in?